Trees Algorithm/binary_tree_traversal_inorder.cpp

```diff
@@ -0,0 +1,21 @@
+//Inorder-> left, root, right
+
+
+#include<bits/stdc++.h>
+using namespace std;
+
+void Inorder( struct node *root)
+{
+  if(root==0)
+  { return 0;}
+ Inorder( root->left);
+ printf("%d", root->data);
+ Inorder( root->right);
+}
```

Comment:
Don't only add the part of the code.
We need a full working code with proper user friendly interface